### PR TITLE
Add Python 3.12 support

### PR DESCRIPTION
**Pull request by Aerylia**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,12 @@
 Changelog
 =========
 
+Version 20.9
+============
+
+* Add Python 3.12 support
```
Comment (kukushechkin) [inline CHANGELOG.rst:8]:
```suggestion
* Added Python 3.12 support `#154 <https://github.com/iqm-finland/iqm-client/pull/154>`_
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+* Support for Python 3.9 will be dropped in a later major release.
```
Comment (kukushechkin) [inline CHANGELOG.rst:9]:
```suggestion
* Python 3.9 support is deprecated and will be removed in the future
```

**Review by kukushechkin — DISMISSED** [review]

**Review by jkotilahti — APPROVED** [review]

**Review by vprusso — APPROVED** [review]
I know my approval likely doesn't count for much as I'm not affiliated with IQM in any way, but as I'm personally interested in getting this PR merged, I did review it, and it does look good to me!

**Comment by Aerylia** [comment]
@vprusso Thanks! Feedback is always appreciated. You need to be an official maintainer for the approval to have an effect on being able to merge. Most of them are currently out with flu, so I'm patiently waiting ^^

**Review by Jajiko — APPROVED** [review]

**Review by jkotilahti — APPROVED** [review]

**Pull request merged**